daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Inputs.scala

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf.transaction
+
+import com.daml.lf.data.Ref
+import com.daml.lf.value.Value.ContractId
+
+import scala.collection.immutable.TreeSet
+
+/** The inputs of the transaction, that is:
+  *   - the contracts were fetched
+  *   - all the parties involved
```

Comment:
What does "involved" mean? The parties that are an informee of some node in the transaction? Or all party IDs mentioned in the transaction? Or something else?